app/Resources/views/themes/demo/layout/header.html.twig

```diff
@@ -7,12 +7,12 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <a class="navbar-brand" href="/"><img src="{{ asset('assets/images/meadow.png') }}" class="img-responsive meadow-logo-top" /></a>
+            <a class="navbar-brand" href="{{ path( ezpublish.rootLocation() ) }}"><img src="{{ asset('assets/images/meadow.png') }}" class="img-responsive meadow-logo-top" /></a>
```

Comment:
According to the [doc](https://github.com/ezsystems/ezplatform-design-engine/blob/master/doc/assets.md) it should be something like that:
```
{{ asset("images/meadow.png", "ezdesign") }}
```
Am I wrong?

Reply:
Indeed, the logo should be themable